pkg/vm/vm_test.go

```diff
@@ -925,6 +925,15 @@ func TestLEFTBadNoString(t *testing.T) {
 	assert.Equal(t, true, vm.state.HasFlag(faultState))
 }
 
+func TestLEFTBadNegativeLen(t *testing.T) {
```

Comment:
nit: what about having tests with params with testdata instead of copy pasting same code for tests?

Reply:
Dunno. At the moment all of these tests are trivial and even though they have some things in common, that's on the order of 3-5 lines which is not a lot to win by making them more complex. And I still hope that #196 will solve most of our VM testing problems.